docs/concepts/artifacts.md

```diff
@@ -96,47 +90,45 @@ from prefect import flow, task
 from prefect.artifacts import create_markdown_artifact
 
 @task
-def my_task():
+def markdown_task():
     na_revenue = 500000
-    markdown_report = """
-        # Sales Report
+    markdown_report = f"""# Sales Report
```

Comment:
next necessarily needed here, but just as an FYI you can use `import textwrap; textwrap.dedent()` on large blocks of text like this to remove all leading whitespace from every line.  Sometimes makes it easier to manage strings like this

Reply:
Thank you!